packages/integrations/src/awslambda.ts

```diff
@@ -0,0 +1,243 @@
+  awsRequestId: string;
+  functionName: string;
+  functionVersion: string;
+  invokedFunctionArn: string;
+  logGroupName: string;
+  logStreamName: string;
+}
+
+interface Module {
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  exports: any;
+  id: string;
+  filename: string;
+  loaded: boolean;
+  parent: Module | null;
+  children: Module[];
+  path: string;
+  paths: string[];
+}
+/**
+ * NodeJS integration
+ *
+ * Provides a mechanism for NodeJS
+ * that raises an exception for handled, unhandled, timeout and similar times of error
+ * and captures the same in Sentry Dashboard
+ */
+export class AWSLambda implements Integration {
+  /**
+   * @inheritDoc
+   */
+  public static id: string = 'AWSLambda';
+
+  /**
+   * @inheritDoc
+   */
+  public name: string = AWSLambda.id;
+
+  /**
+   * context.
+   */
+  private _awsContext: AWSLambdaContext = {} as AWSLambdaContext;
+
+  /**
+   * timeout flag.
+   */
+  private _timeoutWarning?: boolean = false;
+
+  /**
+   * @inheritDoc
+   */
+
+  /**
+   * flush time in milliseconds to set time for flush.
+   */
+  private _flushTimeout?: number = 2000;
+
+  public constructor(options: { context?: AWSLambdaContext; timeoutWarning?: boolean; flushTimeout?: number } = {}) {
+    if (options.context) {
+      this._awsContext = options.context;
+    }
+    if (options.timeoutWarning) {
+      this._timeoutWarning = options.timeoutWarning;
+    }
+    if (options.flushTimeout) {
+      this._flushTimeout = options.flushTimeout;
+    }
+  }
+
+  /**
+   * @inheritDoc
+   */
+  public setupOnce(_: (callback: EventProcessor) => void, getCurrentHub: () => Hub): void {
+    const lambdaBootstrap: Module | undefined = require.main;
+
+    if (!this._awsContext.awsRequestId || !lambdaBootstrap) {
+      return;
+    }
+    /** configured time to timeout error and calculate execution time */
+    const configuredTimeInMilliseconds =
+      this._awsContext.getRemainingTimeInMillis && this._awsContext.getRemainingTimeInMillis();
+
+    /** rapid runtime instance */
+    let rapidRuntime;
+    let originalPostInvocationError = function(): void {
+      return;
+    };
+    if (lambdaBootstrap.children && lambdaBootstrap.children.length) {
+      rapidRuntime = lambdaBootstrap.children[0].exports;
+      /** handler that is invoked in case of unhandled and handled exception */
+      // eslint-disable-next-line @typescript-eslint/no-unsafe-member-access
+      originalPostInvocationError = rapidRuntime.prototype.postInvocationError;
+    }
+
+    const hub = getCurrentHub();
+
+    /**
+     * This function sets Additional Runtime Data which are displayed in Sentry Dashboard
+     * @param scope  - holds additional event information
+     * @hidden
+     */
+    const setAdditionalRuntimeData = (scope: Scope): void => {
+      scope.setContext('Runtime', {
+        Name: 'node',
+        // global.process.version return the version of node
+        Version: global.process.version,
```

Comment:
```suggestion
        name: 'node',
        // global.process.version return the version of node
        version: global.process.version,
```

We usually never capitalize keys, or is there a specific reason doing so?

Reply:
Changes are made in the file but not reflecting.